doc/DeveloperGuide.md

```diff
@@ -88,12 +90,45 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename tag
+
+**MSS**
+
+1. User requests to rename tag
+2. AddressBook shows a list of persons with tag
+3. AddressBook requests user to key in tag to change
+4. User requests the tag to change
+5. AddressBook requests user to key in tag to replace
```

Comment:
Keep in mind that (especially for the project) that the app should have `advanced CLI`.